catkin_ws/src/sim/src/unity_bridge.py

```diff
@@ -61,61 +59,59 @@ def cb_unity_state(msg):
 
     # IMU - NED
     if isIMUActive:
+        pub_imu_sensor_status.publish(Bool(True))
         sbg_quat_msg = SbgEkfQuat()
-        sbg_quat_msg.quaternion = Quaternion(pose_quat_x, pose_quat_y, pose_quat_z, pose_quat_w)
+        q_NED_imunominal = np.quaternion(q_NED_imunominal_w, q_NED_imunominal_x, q_NED_imunominal_y, q_NED_imunominal_z)
+
+        q_NED_imu = q_NED_imunominal * q_imunominal_imu
+        sbg_quat_msg.quaternion = Quaternion(q_NED_imu.x, q_NED_imu.y, q_NED_imu.z, q_NED_imu.w)
         pub_imu_quat_sensor.publish(sbg_quat_msg)
 
         sbg_data_msg = SbgImuData()
-        sbg_data_msg.gyro = Vector3(twist_angular_x[0], twist_angular_y[1], twist_angular_z[2])
+        sbg_data_msg.gyro = Vector3(twist_angular_x, twist_angular_y, twist_angular_z)
         pub_imu_data_sensor.publish(sbg_data_msg)
 
     # DEPTH SENSOR
     if isDepthSensorActive:
+        pub_depth_sensor_status.publish(Bool(True))
         depth_msg = Float64()
```

Comment:
shouldn't this be Float64(-pose_z)?

Reply:
this is the depth sensor status, not the actual depth (I think)

Reply:
`depth_msg = Float64()`?

Reply:
in the line below the fills out the data so its fine

Reply:
I initialize an empty message and then populate it using .data. Its the same as using arguments.